packages/gestalt/src/InternalTextField.js

```diff
@@ -0,0 +1,283 @@
+// @flow strict
+import { useImperativeHandle, useRef, forwardRef, type Element, type Node, useState } from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import Box from './Box.js';
+import TapArea from './TapArea.js';
+import Icon from './Icon.js';
+import focusStyles from './Focus.css';
+import formElement from './FormElement.css';
+import FormErrorMessage from './FormErrorMessage.js';
+import FormHelperText from './FormHelperText.js';
+import FormLabel from './FormLabel.js';
+import Tag from './Tag.js';
+import layout from './Layout.css';
+import styles from './InternalTextField.css';
+import { TAB, ENTER } from './keyCodes.js';
+import typography from './Typography.css';
+
+type Button = 'clear' | 'dropdown';
```

Comment:
Nit: don't think this needs to be pulled out

Reply:
DONE